### PR TITLE
Attach vaccines to programmes directly

### PR DESCRIPTION
**Pull request by thomasleese**
This adds a programme column to vaccines, to change the relationship between programmes and vaccines to be one to many rather than many to many.

### CODE REVIEW

**Review by tvararu — COMMENTED** [review]
Can you also update `lib/tasks/add_health_questions.rake`?

**Comment by thomasleese** [comment]
> Can you also update `lib/tasks/add_health_questions.rake`?

Done, it takes a programme type now instead of a team ID (this might need to change if we decide health questions need to be specific to teams in the future).

File: lib/tasks/add_health_questions.rake [hunk 1]
```diff
@@ -6,24 +6,20 @@ desc <<-DESC
...
-     %i[team_id vaccine_id replace] => :environment do |_task, args|
-  team = Team.find(args[:team_id])
-  vaccine =
-    team
-      .programmes
-      .flat_map(&:vaccines)
-      .find { |v| v.id == args[:vaccine_id].to_i }
+     %i[programme vaccine_id replace] => :environment do |_task, args|
+  programme = Programme.find_by!(type: args[:programme])
+  vaccine = programme.vaccines.find_by(id: args[:vaccine_id])
```
Comment (tvararu) [inline lib/tasks/add_health_questions.rake:22]: I think this breaks as there are references to `team` further down, and the `raise` below needs to be updated too as we don't pass in `team` anymore

Comment (thomasleese) [reply lib/tasks/add_health_questions.rake:22]: Yep, I pushed a little too quickly there, should be working now.

**Pull request merged**